document/core/exec/instructions.rst

```diff
@@ -890,15 +890,15 @@ Memory Instructions
 
    d. Execute the instruction :math:`\MEMORYCOPY`.
 
-   e. Push the value :math:`\vconst_\I32(dst+1)` to the stack.
+   e. Push the value :math:`\vconst_{\I32}(dst+1)` to the stack.
```

Comment:
The style here is inconsistent with e.g. item a in line 885. I don't know which style is the correct one, and if this should be done in this PR, but in the long run I would prefer consistency.

Reply:
Do you mean the Latex style? It's a subscript here but not above. Strictly speaking the braces shouldn't be necessary here either, but our DIY macro expansion working around Sphinx limitations requires it.

Reply:
Oh, one is syntax, the other a meta function, analogous to `t.add` vs `add_t`. Clicking on them should take you to the definitions. They should also appear in different font, although that admittedly isn't all that noticable in the HTML rendering.

Reply:
FWIW, we can probably get rid of the meta function again if we adopt #111.